lib/lndclient/LndClient.ts

```diff
@@ -572,7 +576,16 @@ class LndClient extends SwapClient {
     addHoldInvoiceRequest.setCltvExpiry(cltvExpiry);
     await this.addHoldInvoice(addHoldInvoiceRequest);
     this.logger.debug(`added invoice of ${amount} for ${rHash} with cltvExpiry ${cltvExpiry}`);
-    this.subscribeSingleInvoice(rHash);
+    const SUBSCRIBE_INVOICE_TIMEOUT = 5000;
```

Comment:
I'm wondering whether we need a separate timeout here or whether we can just rely on the existing time limit for the entire swap. Practically speaking I can't think of a reason this should ever take nearly 5 seconds, probably something would be very wrong if so.